config/sync/metatag.metatag_defaults.node.yml

```diff
@@ -0,0 +1,12 @@
+uuid: 470ad0e9-47ab-4346-b8ca-63167db1beca
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: rpwvgyEURXLz_JjgMCrkS1rUv-0k3L79BpO-ReN7fDI
+id: node
+label: Content
+tags:
```

Comment:
Could we include the tags for the header image for basic pages? it might need its own config for that node type.

Reply:
What tag are you looking for?

Reply:
I was thinking at least `og:image`. if i'm not mistaken it should be something like `[node:su_page_banner:entity:su_banner_image:field_media_image:entity:url]`

Reply:
Oh, you want the open graph stuff in there. I left that out as I wasn't sure what our plans on social media integration will be. If we do image we may as well do some of the other ones. I'll see about enabling those.

Reply:
yeah google reads those OG tags too. doesn't it?

Reply:
I'm not sure but I don't see how it could hurt.